common/src/main/java/com/wynntils/core/net/UrlManager.java

```diff
@@ -394,8 +394,6 @@ private InputStream getBundledInputStream() {
     }
 
     private Pair<FileInputStream, File> getLocalCacheInputStreams() {
-        InputStream bundledStream = WynntilsMod.getModResourceAsStream("urls.json");
```

Comment:
?

Reply:
I was checking out whether the manager caches the remote urls incorrectly. It does not. But this input stream was unused, as this method is for loading the local urls (cache), not the bundled one. It was just left there accidentally, so I removed it.